index.js

```diff
@@ -32,6 +32,7 @@ module.exports.pitch = function(remainingRequest) {
 	);
 	this.cacheable && this.cacheable();
 	if(!this.query) throw new Error("query parameter is missing");
+	this._module.userRequest = this._module.userRequest + ' - exposed';
```

Comment:
Could you please add a comment marking this is as a workaround until `module.libIdent()` gets fixed ? :)

Reply:
sure